angular_components/README.md

```diff
@@ -47,10 +47,27 @@ The pub transformer has been removed in favor of code generation through package
   * [angular pub package](https://pub.dartlang.org/packages/angular)
 * [Material Design site](https://material.io)
 
-## Required Icon Font
+## Required Fonts
 
-For icons to appear, add the following icon font download to your page.
-[Example](https://github.com/dart-lang/angular_components_example/blob/ae323a593cbb5508fa6fbe33afb7f5d40ea96a7e/angular_gallery/lib/builder/template/index.html.mustache#L8)
+Add the folowing font downloads to the head element on your page.
+
+### Roboto Font
+
+The typography Sass mixins for are designed to work best with the Roboto font.
```

Comment:
Drop "for" in: The typography Sass mixins <del>for</del> are

Reply:
Level 2 heading above is "**Required** fonts", but the prose here says, "work best with". Is it required or not? :)

If I may suggest a solution here, rather than have level 3 headings under **Required Fonts**, why not just include a bulleted list? Something like this:

## Required Fonts

Add links to the following fonts in the head element of your page:

- **Roboto** ([example](#)):
  ```html
  <link
    rel="stylesheet"
    type="text/css"
    href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700">
  ```
- **Material Icon** ([example](#)):
  ```html
  <link ... >
  ```

Reply:
Good suggestion, thanks!